src/main/java/com/kine/Kine/food/repository/Food.java

```diff
@@ -0,0 +1,57 @@
+package com.kine.Kine.food.repository;
+import com.kine.Kine.food.dto.CreateFoodDTO;
+import com.kine.Kine.food.dto.GetFoodDTO;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+
+@Setter
+@Getter
+@Entity
+@Table(name = "Food")
+public class Food {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @Column(nullable = false)
+    private String name;
+    @Column
+    private float calories;
+    @Column
+    private float proteins;
+    @Column
+    private float carbs;
+    @Column
+    private float fat;
+
+    public Food(String name, float calories, float proteins, float carbs, float fat) {
+        this.name = name;
+        this.calories = calories;
+        this.proteins = proteins;
+        this.carbs = carbs;
+        this.fat = fat;
+    }
+    public Food(long id, String name, float calories, float proteins, float carbs, float fat) {
+        this.id = id;
+        this.name = name;
+        this.calories = calories;
+        this.proteins = proteins;
+        this.carbs = carbs;
+        this.fat = fat;
+    }
+    public Food(){}
+    public GetFoodDTO ConvertToGETFoodDTO(){
```

Comment:
This should be done in the DTO side

Reply:
Is it okay if I made it a private method of service